open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,33 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
+      in: query
+      description:
+        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables) by signaling to the service
+        that they would prefer response to be paginated.
+        
+        Clients will initiate the request by sending an empty pageToken e.g. GET /tables?pageToken or /tables?pageToken=
+        For servers that support pagination, they would recognize pageToken and honor the contracts specified above 
+        by returning a NextPageToken in response if there are more results available.
```

Comment:
for "by returning a NextPageToken in response if there are more results available." can we just move this up as a part of the the contract above?